code_search/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include README.md requirements.txt
```

Comment:
What is this file for?

Reply:
This is a manifest file for the Python package distribution (`sdist`). When the distribution zip is created, these files are included along. `requirements.txt` is needed in `setup.py` to install dependencies (following a standard convention) and `README.md` needed `setup.py` to generate the package long description for `setuptools`.

I find this a convenient way to pass these variables in. Happy to try an alternative approach.